src/__tests__/index.js

```diff
@@ -1,4 +1,57 @@
-const SitemapGenerator = require('../');
+const cheerio = require('cheerio');
```

Comment:
Is cheerio already added? I don't see it in the package json. Between nice work on the tests.

Reply:
It was there as a main dependency already, yeah.